test/schemas.ts

```diff
@@ -1557,9 +1557,24 @@ export const v71Schema = {
     maxSwapSlippagePercentage: 2,
     swapFeeEnabled: false,
     swapFeePercentage: 0.743,
+    createAccountCopyTestType: 'ACCOUNT',
+  },
+}
+
+export const v72Schema = {
+  ...v71Schema,
+  _persist: {
+    ...v71Schema._persist,
+    version: 72,
+  },
+  app: {
+    ...v71Schema.app,
+    maxSwapSlippagePercentage: 2,
+    swapFeeEnabled: false,
+    swapFeePercentage: 0.743,
```

Comment:
Maybe a bad merge?

Reply:
oh my goodness the merge was so bad! i will untangle this mess...

Reply:
thanks for picking up the bad merge @jeanregisser ! i have untangled the mess :D